tests/nodeUnit.js

```diff
@@ -69,7 +69,45 @@ define(function(require) {
           'dist/public/index.html',
           'dist/public/manifest.json',
           'dist/public/offline-worker.js',
-          'dist/public/status.json'
+          'dist/public/status.json',
+          'dist/public/app-manifest.html',
```

Comment:
It's a trade-off. Is the maintenance burden of the test worse than the chance that we might accidentally land a patch that causes some resources to 404?

It's a pretty burdensome test, and when our app was a single page we might have gotten the same test coverage out of a quick glance at the output page. Now that we have a bunch of separate pages it seems like the test is more valuable than before.